appinfo/routes.php

```diff
@@ -81,6 +81,11 @@
 		['name' => 'label#update', 'url' => '/labels/{labelId}', 'verb' => 'PUT'],
 		['name' => 'label#delete', 'url' => '/labels/{labelId}', 'verb' => 'DELETE'],
 
+		// sessions
+		['name' => 'Session#create', 'url' => '/session/create', 'verb' => 'PUT'],
```

Comment:
Would be good if that could be an OCS Api as that makes it possible to be used by external clients and the web UI at the same time.

Steps to do that in summary:
- Move the routes to the ocs section in routes.php
- Move the Controller to extend from OCSController
- Use [generateOcsUrl](https://nextcloud.github.io/nextcloud-router/modules/_index_.html#generateocsurl) in the frontend to build the url for the request
- The response data is wrapped now under `ocs.data` in the JSON response

Reply:
with your instructions it was now straight forward to do this :)

-> https://github.com/nextcloud/deck/pull/3876/commits/9b21cef40d8a67d357d3a773fff04554ddd4dd21